### PR TITLE
Moving OverflowSet to @fluentui/react-internal package

### PR DESCRIPTION
**Pull request by czearing**
#### Pull request checklist
- [x] Part of #15152
- [X] Include a change request file using `$ yarn change`

#### Description of changes
1.	Moving `OverflowSet` from react-next to react-internal (including function component conversion from #13077).
2.	Updating imports for `OverflowSet` in react-internal.
3.	Updating the top level files in react-next to reference @fluentui/react/lib/OverflowSet.
4.	Deleting perf and vr tests for `OverflowSetNext`.
5.	Deleting react-next examples for `OverflowSet`.

### CODE REVIEW

**Comment by ecraig12345** [comment]
Looks like this will require some updates in the charting package to use new props.

File: packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot [hunk 1]
```diff
@@ -827,10 +824,11 @@ exports[`Component Examples renders CommandBar.Basic.Example.tsx correctly 1`] =
             </div>
             <div
               className=
-                  ms-OverflowSet-overflowButton
+                  ms-CommandBar-primaryCommand
```
Comment (ecraig12345) [inline packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot:827]: Do you know why this class name change happened?

Comment (czearing) [reply packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot:830]: Thanks for catching this! getNativeProps was placed incorrectly in the wrapper div and was not applying the className properly.

Comment (ecraig12345) [reply packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot:830]: Nice to see the snapshot tests being useful! :)

File: packages/react-examples/src/charting/AreaChart/AreaChart.Multiple.Example.tsx [hunk 2]
```diff
@@ -197,11 +197,6 @@ export class AreaChartMultipleExample extends React.Component<{}, IAreaChartBasi
             legendsOverflowText={'Overflow Items'}
             yAxisTickFormat={d3.format('$,')}
             legendProps={{
```
Comment (ecraig12345) [inline packages/react-examples/src/charting/AreaChart/AreaChart.Multiple.Example.tsx:199]: Just to confirm, did you check out these examples to verify that they still work as before (even without the FocusZone)? You can compare with the PR deploy site of charting in master. http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/

File: packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot (hunk 1 above)
Comment (ecraig12345) [reply packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot:830]: Hmm original question still stands though--why did the class name change (even after the update to native props location)?

Comment (czearing) [reply packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot]: I updated the pr and it should be good now. The snapshot change was due to `ButtonGrid` applying {...props} after the classNames and then applying and overwriting them.

The remaining snapshot changes for `VerticalBarChart`,`AreaChart`, `OverflowSet`, and `ReizeGroup` are due to the `FocusZone` removal from `OverflowSet` and id updates:

File: packages/react-examples/src/charting/AreaChart/AreaChart.Multiple.Example.tsx (hunk 2 above)
Comment (czearing) [reply packages/react-examples/src/charting/AreaChart/AreaChart.Multiple.Example.tsx:199]: I don't see any behavior changes in the Charting components after the `FocusZone` removal.

File: packages/react-internal/src/components/OverflowSet/OverflowSet.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import '../../common/common';
+@import '~@fluentui/common-styles/dist/sass/common';
```
Comment (ecraig12345) [inline packages/react-internal/src/components/OverflowSet/OverflowSet.scss:1]: I think we can delete this file--I don't see any references to it

**Review by ecraig12345 — APPROVED** [review]
Thanks for all the work on this! I'm going to push an update removing the scss file and then merge it.

**Pull request merged**